go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/filecoin-project/lotus
 
-go 1.13
```

Comment:
@magik6k I'm assuming this is a valid change? The 1.14 is required by a dependency, so I guess this bumps the entire required version to 1.14?